src/c4v/data/baseline_models.py

```diff
@@ -19,10 +19,10 @@
 from skmultilearn.adapt import MLkNN
 from scipy.sparse import lil_matrix
 
-from data_loader import DataLoader
+from c4v.data.data_loader import BratDataLoader
 
 
-class Model_X:
+class Model_Wrapper:
```

Comment:
We have an underscore here. Let's follow the camel case convention of Python for class names.
I remember doing this same comment in a previous PR. Are we sure that we already merge or rebased this on master?

Reply:
This fix is already on master branch. Not sure how to proceed here.

Reply:
I rebased the branch with master and this change has been updated with the correct class name.